cms/models.py

```diff
@@ -87,6 +95,7 @@ class ProgramPage(Page):
         FieldPanel('title_over_image'),
         InlinePanel('courses', label='Program Courses'),
         InlinePanel('faqs', label='Frequently Asked Questions'),
+        FieldPanel('thumbnail_image'),
```

Comment:
I think this should go before the courses and faqs, because these lists can be really long.
Maybe put at the beginning right before the link to the external page?